pkg/components/components.go

```diff
@@ -585,6 +590,11 @@ func GetClusterPermissions() []rbacv1.PolicyRule {
 		},
 		roleWithAllPermissions("coordination.k8s.io", stringListToSlice("leases")),
 		roleWithAllPermissions("route.openshift.io", stringListToSlice("routes")),
+		{
+			APIGroups: stringListToSlice("route.openshift.io"),
+			Resources: stringListToSlice("routes/custom-host"),
+			Verbs:     stringListToSlice("get", "create", "update", "patch"),
```

Comment:
we already have `get` permission on the Route resources, which allows reading all of the route data, so no need `get` also specifically for the `routes/custom-host` subresource.